lemminx-maven/src/test/java/org/eclipse/lemminx/extensions/maven/participants/hover/ManagedVersionHoverTest.java

```diff
@@ -0,0 +1,103 @@
+	public void testManagedVersionHoverInDependencyChiild() throws IOException, URISyntaxException {
+		DOMDocument document = createDOMDocument("/pom-dependencyManagement-child.xml", languageService);
+		Hover hover = languageService.doHover(document, new Position(15, 25), new SharedSettings());
+		String value = ((MarkupContent) hover.getContents().getRight()).getValue();
+		System.out.println("testManagedVersionHoverInDependencyChiild: [" + value + "]");
+		assertNotNull(value);
+		assertTrue(value.contains("The managed version is"));
+		assertTrue(value.contains("2.22.2"));
+		assertTrue(value.contains("The artifact is managed in"));
+		assertTrue(value.contains("org.apache.maven.plugins:maven-surefire-plugin:2.22.2"));
+	}
+
+	@Test
+	@Timeout(90000)
+	public void testManagedVersionHoverInDependencyParent() throws IOException, URISyntaxException {
+		DOMDocument document = createDOMDocument("/pom-dependencyManagement-parent.xml", languageService);
+		Hover hover = languageService.doHover(document, new Position(13, 29), new SharedSettings());
+		String value = ((MarkupContent) hover.getContents().getRight()).getValue();
+		System.out.println("testManagedVersionHoverInDependencyParent: [" + value + "]");
+		assertNotNull(value);
+		assertTrue(value.contains("The managed version is"));
+		assertTrue(value.contains("2.22.2"));
+		assertTrue(value.contains("The artifact is managed in"));
+		assertTrue(value.contains("org.apache.maven.plugins:maven-surefire-plugin:2.22.2"));
+	}
+
+	@Test
+	@Timeout(90000)
+	public void testManagedVersionHoverInPluginChild() throws IOException, URISyntaxException {
+		DOMDocument document = createDOMDocument("/pom-pluginManagement-child.xml", languageService);
+		Hover hover = languageService.doHover(document, new Position(18, 33), new SharedSettings());
+		String value = ((MarkupContent) hover.getContents().getRight()).getValue();
+		System.out.println("testManagedVersionHoverInPluginChild: [" + value + "]");
+		assertNotNull(value);
+		assertTrue(value.contains("The managed version is"));
+		assertTrue(value.contains("2.22.2"));
+		assertTrue(value.contains("The artifact is managed in"));
+		assertTrue(value.contains("org.apache.maven.plugins:maven-surefire-plugin:2.22.2"));
+	}
+
+	@Test
+	@Timeout(90000)
+	public void testManagedVersionHoverInPluginParent() throws IOException, URISyntaxException {
+		DOMDocument document = createDOMDocument("/pom-pluginManagement-parent.xml", languageService);
+		Hover hover = languageService.doHover(document, new Position(14, 33), new SharedSettings());
+		String value = ((MarkupContent) hover.getContents().getRight()).getValue();
+		System.out.println("testManagedVersionHoverInPluginParent: [" + value + "]");
+		assertNotNull(value);
+		assertTrue(value.contains("The managed version is"));
+		assertTrue(value.contains("2.22.2"));
+		assertTrue(value.contains("The artifact is managed in"));
+		assertTrue(value.contains("org.apache.maven.plugins:maven-surefire-plugin:2.22.2"));
```

Comment:
I don't think this should be added in when the version is the one that is defined in the pom. Managed version are only useful when no version is explicitly set (usually in the "child pom")

Reply:
But we still have to show the version and the location links on a pom where the version is clearly defined.

That was actually a problem as the method I've used for finding version in such case was returning a wrong version and link. So, I'd prefer this test case to also exist for parent pom.

Reply:
> But we still have to show the version and the location links on a pom where the version is clearly defined.

we already have that I believe. Ctrl+Click can be used to get the link to the artifact.